WX-877 Update CHANGELOG for release 85 (#7011)
broadinstitute/cromwell: CHANGELOG.md

## 85 Release Notes

### Surface TES System Logs to Cromwell when TES backend returns task error status

> Tes system errors are are now reported in Cromwell execution logs when the TES backend returns a task error.

Reviewer: ```suggestion
TES errors are are now reported in Cromwell execution logs when the TES backend returns a task error.
```

### Only mount reference disks if requested

Reference disks are only mounted if configured in the workflow options.

> ### Support falling back to OCI Manifest Format

Reviewer: ```suggestion
### Support OCI Manifest Format for Docker Registry 
```

Recent docker images of Ubuntu use a new manifest format, this change ensures that these newer image versions can be pulled without issue.

###  Trim down ValueStore logging to prevent OOMs

> ValueStore logging has been reduced, to limit memory usage.

Reviewer: ```suggestion
When converting ValueStore objects to strings for logging, we truncate long values to limit memory usage.
```

### Add support for DRShub

> DRS URL resolution is now performed through the service DRSHub, with Martha as a fallback.

Reviewer: This is Terra-specific and can be left out.

For more information, see the [Cromwell 79 release notes](https://github.com/broadinstitute/cromwell/releases/tag/79).

### TES Improvments

* Surface TES System Logs to Cromwell when TES backend returns task error status

> Tes system errors are are now reported in Cromwell execution logs when the TES backend returns a task error.

Reviewer: This needs a period and TES rendered in all caps. Actually, we can probably just take the first sentence out here since we don't need a headline anymore.
